README.md

```diff
@@ -1,2 +1,185 @@
-# hello-microverse
-Microverse first project
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] Kindly remove this logo or customise it if you have one. This will make your repo to be more professional. 👍